### DISCUSSION TITLE
Mistake in the playground?

### DISCUSSION THREAD

**Discussion by jeroenvermunt** [modular/modular#283] [General]
I just started the playground, and noticed this:

![image](https://github.com/modularml/mojo/assets/39826623/95717f4a-9eb5-4544-ba0a-813fb113185b)

I uncommented the `uncomment to see and error` part, but there is not any error appearing, am I misinterpreting the documentation?

**Comment by abhinav-upadhyay** [comment]
I'm seeing error on uncommenting that line. 
![Screenshot from 2023-05-29 15-53-50](https://github.com/modularml/mojo/assets/328300/668e48d6-31e0-4719-a0d0-c5d1b9b18932)

**Reply by jeroenvermunt** [reply]
Thanks for the quick reply!

I see the difference, apparently I removed the `let` declaration in `let c = a` and forgot that I had done so.

**Comment by mfranzon** [comment]
I think that the problem should be the kernel. If you run the correct code, than you re-run the same cell with the wrong statement seems that it still working due to the cache of the main, from my understanding. 
@jeroenvermunt if you try to rerun the kernel and than run the cell all should work fine and the error should be raised:)
